questionnaire/2020/questions.yaml

```diff
@@ -215,6 +215,20 @@
     - A little
     - A lot
 
+  - title: What connection methods / transports are being utilized as part of your network automation implementation?
+    type: Multiple choice 
+    id: transport-protocol
+    responses:
+    - CLI
+    - Netconf
+    - Restconf
+    - gNMI
+    - gRPC
+    - HTTP/HTTPS
+    - Console
```

Comment:
Should we specify "Serial console"?

Reply:
Isn't console still just SSH? Is that a transport?

Reply:
Good point @abenokraitis . We could group CLI  / ssh / console under a single response.

Reply:
I think Console should be its own item and agreed with rename it `Serial Console`
Pyez from juniper is able to run automation over the console natively and its extremely useful. I think pyATS support that too and you can also use netmiko for that.